1 HW_Standard/HW_Gabite.yaml

```diff
@@ -0,0 +1,21 @@
+species: Gabite
+ingamename: R. Rotten
+displayname: Robbie Rotten
+setname: Halloween
+tags: [halloween]
+item: Nanab Berry
+ability: Color Change
+moves:
+    - [Feint Attack]
+    - [Double Team]
+    - [Spider Web]
+    - [Slack Off]
+happiness: 255
+biddable: True
+rarity: 1.0
+ball: [Net]
+shiny: True
+nature: Lax
+ivs: {hp: 31, atk: 31, def: 31, spA: 31, spD: 31, spe: 0}
+evs: {hp: 0, atk: 200, def: 72, spA: 0, spD: 0, spe: 0}

```

Comment:
Given the weak mono-attack and healing, i would consider moving some investment into attack